README.md

```diff
@@ -46,42 +46,11 @@ For new projects, the best way to start is by using [Spin](https://github.com/tm
     ```
 
 3. Provision React.js library
-
-    `jsoo-react` does not make any assumptions about how you will load React.js in your application. There are 2 ways of doing so:
-
-    #### With Webpack (or any JavaScript bundler)
-
-    If you want to use Webpack, Rollup, Parcel or any other JavaScript bundler, include a file `react-requires.js` in your application source folder with the following content:
-
-    ```js
-    joo_global_object.React = require('react');
-    joo_global_object.ReactDOM = require('react-dom');
-    ```
-
-    Then add it to your application `dune` file so it can be linked:
-
-    ```
-    (executables
-    ...
-    (js_of_ocaml
-      (javascript_files react-requires.js)))
-    ```
-
-    To see an example of this approach, check the [example](example) folder.
+    
+    `jsoo-react` uses `require` to import React and ReactDOM. This means that you will likely need to use a bundler such as Webpack or rollup.js.
 
     Note that at this moment, `jsoo-react` is compatible with **React 16**, so be sure to have the appropriate constraints in your `package.json`.
 
-    #### With `<script>` tags
```

Comment:
Hm right, it's a bit of a pity that we lose this possibility. But I guess it's fine? The only app out there that I know of uses webpack: https://github.com/michael-schwarz/goblint-webfrontend-jsoo/blob/2d88e9649078d0138ff5f1e76e2391f7cd684e01/gobview/webpack.config.js

cc @davesnx @michael-schwarz @keremc

Reply:
Yeah that's what I meant yesterday when I said this would be forcing people to use a bundler, but I don't think it's a horrible trade-off as most people bundle React that way. 

The one way to get around that would be to functorize the interface that produces the React bindings, but I think that might require more refactoring for another PR.

Reply:
I agree. I wouldn't add the functor though until there's someone with a use case that requires it. :)

Reply:
Yup, we indeed use webpack. 
It's really exciting to see all the recent work y'all have been doing with `jsoo-react`, thank you! :tada: 

(Btw, we once again moved the project to a new (now permanent) repo within our GitHub organization https://github.com/goblint/gobview)